tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -410,12 +411,12 @@ def test_has_issue(self):
         assert len(response.data["data"]) == 1
         assert response.data["data"][0]["issue"] == event.group.qualified_short_id
 
-        # should only show 1 event of type transaction since they dont have issues
-        query = {"field": ["project", "issue"], "query": "!has:issue", "statsPeriod": "14d"}
-        response = self.do_request(query, features=features)
-        assert response.status_code == 200, response.content
-        assert len(response.data["data"]) == 1
-        assert response.data["data"][0]["issue"] == "unknown"
+        # # should only show 1 event of type transaction since they dont have issues
+        # query = {"field": ["project", "issue"], "query": "!has:issue", "statsPeriod": "14d"}
+        # response = self.do_request(query, features=features)
+        # assert response.status_code == 200, response.content
+        # assert len(response.data["data"]) == 1
+        # assert response.data["data"][0]["issue"] == "unknown"
```

Comment:
Let's just delete this instead of commenting it out

Reply:
Will do, yeah wasn't sure if we wanted to resurrect later.